virtool/http/auth.py

```diff
@@ -20,6 +26,14 @@
     "permissions"
 ]
 
+UNAUTHENTICATED_ROUTES = [
+    "/api/account/login",
+    "/api/account/logout",
+    "/api/acquire_tokens",
+    "/api/refresh_tokens",
+    "/api/delete_tokens"
+]
```

Comment:
Can be replaced with `public` kwarg on route decorators. See comment further down.

BTW: Use `tuple` for constants that are [sequences](https://docs.python.org/3/library/stdtypes.html). They are immutable, unlike lists.